javatests/google/registry/flows/EppFlowInjectionModule.java

```diff
@@ -0,0 +1,46 @@
+package google.registry.flows;
+
+import org.joda.time.DateTime;
+
+import dagger.Module;
+import dagger.Provides;
+import google.registry.flows.EppException.UnimplementedOptionException;
+import google.registry.flows.domain.ExtraDomainValidation;
+import google.registry.model.domain.launch.LaunchCreateExtension;
+import google.registry.model.registry.label.ReservationType;
+
+@Module
+public class EppFlowInjectionModule {
```

Comment:
All public classes (and their public members) should have a javadoc summary fragment. Relevant section of style guide:

https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required